packages/plans-grid-next/src/components/shared/action-button/style.scss

```diff
@@ -0,0 +1,26 @@
+.plans-grid-next-action-button {
+	.plans-grid-next-comparison-grid & {
+		margin-bottom: auto;
+
+		.plans-grid-next-action-button__content {
+			min-height: 40px;
+		}
+	}
+
+	.plan-features-2023-grid__plan-spotlight & {
+		width: 200px;
+	}
```

Comment:
If we agree, then we can start already by defining what the default styling is (to keep here) and move the customizing/overriding to the respective stylesheets for "spotlight" and "comparison grid". So no longer using parent selectors here.

I think there are pros/cons - but the more general the component I think the more it makes sense to isolate further.

Reply:
👍 Updated: [dea08fe](https://github.com/Automattic/wp-calypso/pull/93312/commits/dea08fe3d55673ef989f80521424f2cf5f4d52be)